.github/actions/verify-token/src/predicate1.ts

```diff
@@ -81,6 +81,7 @@ export async function createPredicate(
       },
       resolvedDependencies: [
         {
+          name: "sourceRepo",
```

Comment:
is this a standard annotation in the specs?

Reply:
Sort of, it can be:

```
Machine-readable identifier for distinguishing between descriptors.

The semantics are up to the producer and consumer. The name SHOULD be stable, such as a filename, to allow consumers to reliably use the name as part of their policy.
```

So `sourceRepo` can distinguish between descriptors, and can allow verifiers to check against policy. It is not however the filename convention mentioned in there

Reply:
It's not super important so we could just leave it off. The rest of the change is still valid.

Reply:
I vote for not adding it and see if we need it in the future.
The intoto specs gives us the ability, but it'd be nice if SLSA standardized some of the common names, and give some guidance about how it's supposed to be used @kpk47

Reply:
Yep. Already removed it.